website/docs/advanced-concepts/granular-access-control/README.md

```diff
@@ -19,61 +19,43 @@ tags={[
-- To manage user roles and permissions, go to **Admin Settings** > **Access Control** > **Users**. It displays a list of all users with their roles and groups assigned to them. You can add a new user to your instance by clicking on **Add Users** at the top right corner of the screen.
-
-<ZoomImage
-  src="/img/GAC_users.png" 
-  alt="Add a new Role"
-  caption="Users' Window"
-/>
-
-## Groups
-
-A group is a collection of users that allows you to assign roles to multiple users at once, simplifying permission management. Here are some important things to know about groups:
-
-- You can assign multiple roles to a group, and users can belong to multiple groups.
-- To create a new group, navigate to **Admin Settings** > **Access Control** > **Groups**, and click the **Add group** button in the upper-right corner of the screen.
-- To manage groups and the roles assigned to them, go to **Admin Settings** > **Access Control** > **Groups**. This page shows all the groups, and clicking on one displays the users who are members of the group, along with the roles assigned to them.
-
-When you share an application or workspace, you can invite a group and assign the desired role to it. This makes it easy to grant permissions to multiple users at once, and you can modify the group's roles to update the permissions for all its members simultaneously.
-
-
-## Programmatic access control
-
-You can utilize roles and groups assigned to a user to programmatically manage access to various entities such as widgets, datasources, APIs, and queries. 
-
-To enable Programmatic access control, follow the steps below - 
-
-1. On the homepage, go to **Admin Settings > General**.
-2. Under the **Programmatic Access Control** section, select the **Access roles and user groups in code for conditional business logic** checkbox. 
-
-<ZoomImage
-  src="/img/Enable_programmatic-access-control.png" 
-  alt="Programmatic Access Control"
-  caption="Enable Programmatic access control"
-/>
-
-3. To manage access control programmatically, use the relevant properties of the Appsmith user object `appsmith.user.roles` or  `appsmith.user.groups` in JavaScript code to obtain an array of roles or groups assigned to the logged-in user. For more information, refer to the [roles](/reference/appsmith-framework/context-object#roles) and [groups](/reference/appsmith-framework/context-object#groups) properties of the Appsmith user object.
-
-## Further reading
+Granular Access Control (GAC) in Appsmith allows you to specify precise access permissions for every entity within your Appsmith instance. This includes applications, users, workspaces, queries, and more. Here’s a quick overview of how GAC works:
+
+  <ZoomImage
+   src="/img/gac-overview.png" 
+   alt="How Granular Access Control works in Appsmith"
```

Comment:
Can we use a simplified image like this, this is too complex to grok. Let's simplify the principles for our users.
![GAC-overview](https://github.com/appsmithorg/appsmith-docs/assets/16632903/0efdd84f-b8c2-413b-97b3-ab0ca512f510)

Reply:
Some of the user tests we ran - had no issues in understanding the image. I will run more tests and based on the results will take it ahead.

Reply:
Deferred for now.

Reply:
@jnikhila  - Can you share these user tests? Are these external ones or internal solution team ones?